src/ResourceManager/IotCentral/Commands.IotCentral/help/Set-AzureRmIotCentralApp.md

```diff
@@ -0,0 +1,231 @@
+---
+external help file: Microsoft.Azure.Commands.IotCentral.dll-Help.xml
+Module Name: AzureRM.IotCentral
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.iotcentral/set-azurermiotcentralapp
+schema: 2.0.0
+---
+
+# Set-AzureRmIotCentralApp
+
+## SYNOPSIS
+Updates the metadata for an IoT Central Application.
+
+## SYNTAX
+
+### ResourceIdParameterSet (Default)
+```
+Set-AzureRmIotCentralApp [-DisplayName <String>] [-Tag <Hashtable>] -ResourceId <String> [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### InputObjectParameterSet
+```
+Set-AzureRmIotCentralApp [-DisplayName <String>] [-Tag <Hashtable>] -InputObject <PSIotCentralApp> [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### InteractiveIotCentralParameterSet
+```
+Set-AzureRmIotCentralApp [-DisplayName <String>] [-Tag <Hashtable>] [-ResourceGroupName] <String>
+ [-Name] <String> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Update the metadata for an IoT Central Application. 
+
+## EXAMPLES
+
+### Example 1 Update Display Name
+```powershell
+PS C:\> Set-AzureRmIotCentralApp -ResourceGroupName "MyResourceGroupName" -Name "MyAppResourceName" -DisplayName "My New Custom Display Name"
+```
+
+Update the Display name on an existing IoT Central Application.
+
+Example Output:
```

Comment:
Default output display should be table format, see: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/generate-format.ps1xml-file.md this applies across the board, for all cmdlets